.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Should this not be in your global gitignore?

Reply:
I guess I forgot. But I also don't mind to have this protection per repo tbh.
I'm also sorry, did that work on the Mac and they still do this stuff every once in while. :sigh: